source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTableView.mm

```diff
@@ -52,15 +57,20 @@ - (void)defineColumnDefinitions:(const std::shared_ptr<Table> &)table
 
     for (auto columnDefinition : table->GetColumns()) {
         auto optionalNumericValue = columnDefinition->GetWidth();
+        ACRColumnDefinition *iOSColumnDefinition = nil;
         if (optionalNumericValue.has_value()) {
-            [_columnDefinitions addObject:[[ACRColumnDefinition alloc] initWithRelativeWidth:optionalNumericValue.value_or(1) / totalRelativeWidth
-                                                                             totalPixelWidth:totalPixelWidth]];
+            iOSColumnDefinition = [[ACRColumnDefinition alloc] initWithRelativeWidth:optionalNumericValue.value_or(1) / totalRelativeWidth
```

Comment:
should probably check to make sure `totalRelativeWidth > 0`

Reply:
because of the valuer_or(1), the total relative width will be bigger than 0, and the total relative width value will be used only if the optional numeric value has a value. But I agree with you, the codes can change, and could be hard to maintain this relationship.